src/index.ts

```diff
@@ -129,6 +134,20 @@ function sanitize (str: string): string {
   return str.replace(/\u0026/g, '&amp;').replace(/\u003C/g, '&lt;')
 }
 
+function sanitizeAttribute (str: string): string {
+  return str.replace(/\u0022/g, '\"')
+}
+
+interface AccessibleTextProps {
+  status: string;
+  label?: string;
+}
+
+function createAccessibleText({label, status}: AccessibleTextProps): string {
+  const labelPrefix = label ? `${label}: ` : '';
+  return labelPrefix + status;
```

Comment:
The accessible text does not include icons or colors due to their added verbosity, though if icons or colors tend to be used in a critical manner, this can be easily changed.